pennylane/ops/qubit/parametric_ops.py

```diff
@@ -119,6 +119,10 @@ def _controlled(self, wire):
         new_op = CRX(*self.parameters, wires=wire + self.wires)
         return new_op.inv() if self.inverse else new_op
 
+    def simplify(self):
+        theta = qml.math.mod(self.data[0], 4 * np.pi)
+        return RX(theta, wires=self.wires)
```

Comment:
The `mod` base for some of these operations is `4 * np.pi` instead of `2 * np.pi` because of the -1 global phase introduced. It looks like we treat global phase with significance (e.g. due to the presence of both `qml.RZ` and `qml.PhaseShift`).

Thoughts?

Reply:
Yep... Let's use `4*np.pi` then.  The negative sign will matter in controlled versions of the operation.

Reply:
@eddddddy could you elaborate more on this please?

Reply:
This is explained by @albi3ro's comment below. Let me know if you need more clarification.

And yes, I think we agreed that we'll handle the modulo the way NumPy does.